server/src/bots/telegram/services/messageHandlerRegistry.ts

```diff
@@ -190,6 +190,39 @@ export const withSingleParameterAfterCommand = (
     };
 };
 
+export const withTwoArgumentsAfterCommand = (
```

Comment:
if you could integrate it with the previous approach it would be great
and it would be great if it were generic 

```

handlerFn.call(
                this,
                bot,
                message,
                chatId,
                ...parameters
            );

```

Reply:
let me know if you want to brainstorm on that together

Reply:
also, be aware, that from now these FNs have there own files (because of testing framework)

Reply:
Brainstorm ***IS COMMING***

Reply:
done